Utils/utils.py

```
@@ -47,36 +47,52 @@ class CmdData:
+                int: re.compile(fr'-{key}\s+(-?\d+)\b'),
+                float: re.compile(fr'-{key}\s+(-?\d+\.\d+)\b')
+            }[typ]
+
+            match = pattern.search(remaining)
+            if not match:
+                continue
 
-    params, args = raw, {}
+            try:
+                val = True if typ is bool else typ(match.group(1).strip())
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)